lib/active_set/paginating/operation.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+class ActiveSet
+  module Paginating
+    class Operation
+      def initialize(set, instructions_hash)
+        @set = set
+        @instructions_hash = instructions_hash
+      end
+
+      def execute
+        [ActiveRecordStrategy, EnumerableStrategy].each do |strategy|
+          maybe_set_or_false = strategy.new(@set, operation_instructions).execute
+          break(maybe_set_or_false) if maybe_set_or_false
+        end
+      end
+
+      def operation_instructions
+        @instructions_hash.symbolize_keys.tap do |h|
```

Comment:
This handles setting defaults. It also ensures that a `count` is set for the paginating instructions. This is important, as it will allow any outside inspector (like `ActionSet`) to know how large the unpaginated set is.